pkg/storaged/utils.js

```diff
@@ -749,7 +749,7 @@ export function teardown_active_usage(client, usage) {
             // If we would remove all physical volumes of a volume
             // group, remove the whole volume group instead.
             if (pvs.length == client.vgroups_pvols[p].length) {
-                return vg.Delete({ 'tear-down': { t: 'b', v: true } });
+                return vg.Delete({ 'tear-down': { t: 'b', v: true } }).then(reload_systemd);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220914-124338-048b9d83-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)